src/gitpodServer.ts

```diff
@@ -20,6 +20,11 @@ interface ExchangeTokenResponse {
 	scope: string;
 }
 
+interface GrantedToken {
+	token: string;
+	scopes: string;
+}
+
```

Comment:
I'll revert this if server changes are not needed, see https://github.com/gitpod-io/gitpod-vscode-desktop/pull/17/files#r956620417